src/http-jsonp-plugin.js

```diff
@@ -0,0 +1,98 @@
+//@flow
+
+import {Utils} from 'playkit-js'
+import {Error} from 'playkit-js'
+
+/**
+ *  A networking plugin to handle http and https URIs via XHR and for playmanifest using jsonp
+ * @param {string} uri - url pf the request
+ * @param {Object} request - request object
+ * @param {Function} callback - external redirect callback function
+ * @return {Promise<*>} - a promise with the response
+ * @export
+ */
+export default function (uri: string, request: Object, callback: Function): Promise<*> {
+  const _executeXHR = function (uri, request) {
```

Comment:
@OrenMe  why not to use the `Utils.Http.execute(...)` instead of re-write all this logic again?